packages/sanity/src/core/bundles/util/util.ts

```diff
@@ -1,17 +1,24 @@
 import speakingurl from 'speakingurl'
 
 import {type BundleDocument} from '../../store/bundles/types'
+import {isVersionId} from '../../util'
 
 const PUBLISHED_SLUG = 'Published'
+const DRAFTS_SLUG = 'Drafts'
 
 /**
  * @internal
  * @hidden
  */
 export function getBundleSlug(documentId: string): string {
   if (documentId.indexOf('.') === -1) return PUBLISHED_SLUG
-  const version = documentId.slice(0, documentId.indexOf('.'))
-  return version
+
+  if (isVersionId(documentId)) {
+    const [_, bundleSlug, ...PublishedId] = documentId.split('.')
+    return bundleSlug
+  }
+
+  return DRAFTS_SLUG
```

Comment:
same as comment above - I think we should rather return undefined in this case too.

Reply:
Done here: https://github.com/sanity-io/sanity/pull/7357/commits/a362dee82bb47cef9ad3404d75000dd79759d107